src/modules/PowerShell/FinOpsHub/FinOpsHub.psd1

```diff
@@ -0,0 +1,141 @@
+#
+# Module manifest for module 'FinOpsHub'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 4/17/2023
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'FinOpsHub.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
```

Comment:
Should this version match the release version? As in, even if the PS module hasn't changed, update it to 2.0 to match the Toolkit v2.0? The value in that is ensuring you're always using the right version instead of knowing that you need PS module v1.3.4 to work with Toolkit v2.1.8. If we do want to align this, do we have a good way to use a centrally-managed version number? Not a huge concern, but just a thought for the future.

Reply:
It depends on how tightly we align the functionality I guess. One thing we can do is set this version at release time.

Reply:
I like that. Will mark this as resolved for now, but will be good to keep in the back of our minds as we move forward.